packages/esm-patient-chart-app/src/routes.json

```diff
@@ -131,7 +131,7 @@
       "slot": "start-visit-workspace-form-slot",
       "component": "startVisitForm",
       "meta": {
-        "title": "Start a visit"
+        "title": "startVisit"
```

Comment:
The only reason this translation is working is because the translation for this key `startVisit` is defined at multiple places, one of which is [here](https://github.com/openmrs/openmrs-esm-patient-chart/blob/0aff26e7faba9f6d4c673745ba26bbfa40f3361c/packages/esm-patient-chart-app/src/visit/start-visit-button.component.tsx#L21), hence somehow indirectly its working on our behalf.

To properly define the translation of the title, you'll need to specify the translation in a comment above the workspace definition in the `index.ts`.

Reply:
Yeah, and that should mean you can undo the change to this line.